util/src/main/java/org/folio/reservoir/util/readstream/Marc4ParserToJson.java

```diff
@@ -0,0 +1,75 @@
+package org.folio.reservoir.util.readstream;
+
+import io.vertx.core.Handler;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.streams.ReadStream;
+import java.io.ByteArrayOutputStream;
+import org.marc4j.MarcJsonWriter;
+import org.marc4j.converter.impl.AnselToUnicode;
+
+public class Marc4ParserToJson implements ReadStream<JsonObject> {
```

Comment:
Should be Marc4jParserToJson

Reply:
This parser is not bound to Marc4j unlike the other sister parser which produces a Marc4j Record.

Reply:
Then let's call it just "MarcParserToJson" or better "MarcToJsonParser"

Reply:
Sure, that's fine. If I ever replace marc4j the MarcToJsonParser will have same interface and the marc4j one goes away.